<?xml version="1.0"?>
<!-- vim: set fenc=utf-8 et sts=4 ft=xml : -->
<launch>

  <node
    machine="navigate"
    name="compute_sun_angle"
    pkg="sun_pointing"
    type="compute_angle_server.py"
    ns="/processes/compute_sun_angle">

    <param name="spin_velocity" value="0.3"/>
    <param name="odometry_frame" value="odom"/>
    <param name="yaw_update_period" value="0.05"/>

    <remap from="image" to="/cameras/manipulator/left/rect_color"/>
    <remap from="simple_move" to="/processes/simple_move"/>
    <remap from="twist" to="/motion/planner_command"/>

</node>
</launch>
